src/views/components/share-twitter-button.js

```diff
@@ -0,0 +1,21 @@
+import { h } from 'preact' /** @jsx h */
+
+export default ({ href, text, className }) => {
+  const handleClick = () => {
+    window.open(
+      `https://twitter.com/intent/tweet?text=${encodeURIComponent(text)}&url=${href}`,
```

Comment:
Pro twitter, achei essa pergunta no SO, que resolveu meu problema.
http://stackoverflow.com/questions/10424275/how-can-i-open-a-twitter-tweet-using-the-native-twitter-app-on-ios

Não sei nem por qual motivo essa resposta tá negativada.. Mas enfim, resolveu pra mim..